subsys/bluetooth/controller/include/ll_feat.h

```diff
@@ -189,10 +189,20 @@
 #define LL_FEAT_BIT_CIS_PERIPHERAL 0
 #endif /* !CONFIG_BT_CTLR_PERIPHERAL_ISO */
 
+#if defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_CONN_ISO)
+#if defined(CONFIG_BT_ISO_TX_MTU)
+#define BT_CTLR_ISO_TX_BUFFER_SIZE MIN((CONFIG_BT_ISO_TX_MTU + 8U), \
+				       MAX(CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE, \
+					   8U))
```

Comment:
Minimum value of `9U` will ensure a minimum SDU length of `1U`.

```suggestion
#define BT_CTLR_ISO_TX_BUFFER_SIZE MIN((CONFIG_BT_ISO_TX_MTU + 8U), \
				       MAX(CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE, \
					   9U))
```